ui/src/message_popup/author/edit_question_page.jsx

```diff
@@ -14,12 +16,45 @@ export default React.createClass({
     questionId: React.PropTypes.string.isRequired
   },
 
+  getInitialState(){
+    return ({
+      allDatabaseQuestions: {
+        currentQuestions: [],
+        archivedQuestions: []
+      }
+    });
+  },
+
+  componentDidMount(){
+    Api.questionsQuery().end(this.onQuestionsReceived);
+  },
+
+  onQuestionsReceived(err, response){
+    const allDatabaseQuestions = JSON.parse(response.text).row;
```

Comment:
I don't quite follow what's happening here (same comments as in `questions_page.jsx`).